src/extension.ts

```diff
@@ -109,9 +112,9 @@ async function doActivate(_operationId: string, context: vscode.ExtensionContext
 
     // debug
     registerCommand(context, "maven.plugin.debug", debugHandler);
-    vscode.debug.onDidTerminateDebugSession((session: any) => {
+    vscode.debug.onDidTerminateDebugSession((session) => {
         if (session.type === "java") {
-            const terminalName: string = session._configuration.terminalName;
+            const terminalName: string = session.configuration.terminalName;
```

Comment:
data structure of `session` changes?

Reply:
> data structure of `session` changes?

'configuration' is the get method of '_configuration' and it is provided in the type definition, so I used it instead.